Review chapter two notes (#3)
annemarie35/python-training: README.md

# Computer-science basics

This repository is aiming to pin down basics computer science concepts.

## Sources

### MIT CS introductory course

#### 1984 - 2007

> This course, designed by Harold Abelson and Gerald Jay Sussman, use Lisp.

Reviewer: Where did you find this information ?

#### 1984 - 2007

This course, designed by [Harold Abelson](https://en.wikipedia.org/wiki/Structure_and_Interpretation_of_Computer_Programs) and [Gerald Jay Sussman](https://en.wikipedia.org/wiki/Gerald_Jay_Sussman), use Lisp.

Its foreword, written by [Alan Perlis](https://en.wikipedia.org/wiki/Alan_Perlis), clearly states that most of the time in computer science should be devoted to understanding the problem and finding a way to attack it using unambiguous method, not to write source code. 

> Educator, generals, dieteticians, psychologists, and parents program. Armies, students, and some societies are programmed. An assault on large problems employs a succession of programs, most of
which spring into existence en route. ese programs are rife with issues that appear to be particular to the problem at hand. To appreciate programming as an intellectual activity in its own right you must turn to computer programming/
(...)

> > Every computer program is a model, hatched in the mind, of a real or mental process. ese processes, arising from human experience and thought, are huge in number, intricate in detail, and at any time only partially understood. ey are modeled to our permanent satisfaction rarely by our computer programs.

Reviewer: ```suggestion
These processes, arising from human experience and thought, are huge in number, intricate in detail, and at any time only partially understood. They are modeled to our permanent satisfaction rarely by our computer programs.
```
